bbn-test-4/finality-providers/registry/Bevm.json

```diff
@@ -0,0 +1,15 @@
+{
+  "description": {
+    "moniker": "Bevm",
+    "identity": "",
+    "website": "https://bevm.io",
+    "security_contact": "",
```

Comment:
Thanks! I would appreciate it if you could specify a contact and recreate the signature.